components/dashboard/src/Menu.tsx

```diff
@@ -110,6 +110,7 @@ export default function Menu() {
         "keys",
         "integrations",
         "preferences",
+        "usage",
```

Comment:
Nit: I personally find it easier when the order of URLs is roughly aligned with the order in menus 😇 would you mind adding `"usage"` right below `"billing"`? 👀

Reply:
Good point! 🙂 
Moved and pushed again in cc460a6.

Thanks for taking a look, @jankeromnes! 🏀